src/config.js

```diff
@@ -50,3 +50,10 @@ export const PLAUSIBLE_DOMAIN = process.env.REACT_APP_PLAUSIBLE_DOMAIN;
 
 export const LANDSCAPE_PROFILE_IMAGE_MAX_SIZE =
   process.env.REACT_APP_LANDSCAPE_PROFILE_IMAGE_MAX_SIZE || 10000000; // 10 MB
+
+export const AGRICULTURAL_PRODUCTION_METHOD_LIVELIHOODS = process.env
+  .REACT_APP_AGRICULTURAL_PRODUCTION_METHOD_LIVELIHOODS || [
+  'Industry',
+  'Fishing',
+  'Crop farming',
+];
```

Comment:
suggestion: pull these from a strings file or use placeholder text, i.e. `LIVELIHOOD_EXAMPLE`

Reply:
@paulschreiber These are the specific 3 values for livelihoods that if selected trigger the display of the agricultural production method field, how is that related to a placeholder text? or why should this be inside a strings file?

Reply:
Got it. Please leave a comment to that effect.